numpy_challenge.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+
+# create array by 3 different ways
+a = np.array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
+b = np.arange(1, 11)
+c = np.random.rand(10)
+
+
+def matrix_multiplication(A, B):
+    res = np.zeros((np.shape(A)[0], np.shape(B)[1]))
+
+    for i in range(np.shape(A)[0]):
+
+        for j in range(np.shape(B)[1]):
+
+            for k in range(np.shape(B)[0]):
+                res[i][j] += A[i][k] * B[k][j]
```

Comment:
Вот это велосипедище) Лучше было использовать библиотечную функцию, мы же всё таки нампай проходим. Можно было и циклами, но тогда хотя бы использовать векторизацию
```suggestion
    res = np.zeros((np.shape(A)[0], np.shape(B)[1]))

    for i in range(np.shape(A)[0]):

        for j in range(np.shape(B)[1]):
                res[i][j] += np.sum(A[i, :] * B[:, j])
```